components/providers/AppBridgeProvider.jsx

```diff
@@ -43,18 +43,18 @@ export function AppBridgeProvider({ children }) {
 
     return {
       host,
-      apiKey: process.env.SHOPIFY_API_KEY,
+      apiKey: process.env.SHOPIFY_APP_API_KEY,
```

Comment:
do we need to do `SHOPIFY_APP_API_KEY || SHOPIFY_API_KEY`, so this works with older versions of the CLI?

Reply:
Yes, I followed this pattern pretty much everywhere so that the new template also works with the current version of the CLI - in case someone creates a new app without first updating their CLI.